pepper-apis/config/tcell_agent.config.ctmpl

```diff
@@ -0,0 +1,27 @@
+
+{{with $environment := env "ENVIRONMENT"}}
+{{with $conf := vault (printf "secret/pepper/%s/tcell" $environment)}}
+{{with $tcell := index $conf.Data}}
+
+{
+  "version": 1,
+  "applications": [
+    {
+      "host_identifier": "study server backend",
```

Comment:
Would be great to set `host_identifier` as an env var so we can keep housekeeping and server straight in tcell dashboard.

Reply:
Was going to ask about confusing housekeeping and back-end but looks like you have concluded mixing the two is acceptable for now?

Reply:
I'm going to try setting env vars and see if that does the trick...